api-lib/gql/__generated__/zeus/index.ts

```diff
@@ -323,7 +323,7 @@ export class GraphQLError extends Error {
   constructor(public response: GraphQLResponse) {
     super('');
     // eslint-disable-next-line no-console
-    console.info(JSON.stringify(response, null, 2));
+    console.info(JSON.stringify(response));
```

Comment:
how do we fix this permanently? I thought our script did it before

Reply:
@CryptoGraffe and @levity does this line work on your machines? https://github.com/coordinape/coordinape/blob/dadae6d9eaacc42f4e88d2312bbf534dbad69454/scripts/generate_zeus.sh#L43

Reply:
@CryptoGraffe fixed here: https://github.com/coordinape/coordinape/pull/1455/commits/64df0c18a57ce2ffc098e796890086a7e5bab0f5